include/exadg/operators/operator_base.h

```diff
@@ -106,7 +106,8 @@ class OperatorBase : public dealii::Subscriptor
 
   static unsigned int const vectorization_length = dealii::VectorizedArray<Number>::size();
 
-  typedef std::vector<dealii::LAPACKFullMatrix<Number>> BlockMatrix;
+  typedef dealii::LAPACKFullMatrix<Number> CellMatrix;
+  typedef std::vector<CellMatrix>          BlockMatrix;
```

Comment:
should be `BlockMatrices` in my opinion, or maybe we should always write explicitly `std::vector<CellMatrix>` where needed for clarity.

Reply:
Done in 04e1da7b41495fabec19ac2807737fb101f1726f

I decided to keep a typedef for `LAPACKMatrix` to shorten the name without changing the meaning. The std::vector reflects everywhere that we simply have a vector of LAPACK `matrices`.